docs/data/toolpad/core/components/dashboard-layout/DashboardLayoutBranding.tsx

```diff
@@ -37,19 +49,20 @@ export default function DashboardLayoutBranding() {
   }, [pathname]);
 
   return (
-    <AppProvider navigation={NAVIGATION} branding={BRANDING} router={router}>
+    // preview-start
+    <AppProvider
+      navigation={NAVIGATION}
+      branding={{
+        logo: <img src="https://mui.com/static/logo.png" alt="MUI logo" />,
+        title: 'MUI',
+      }}
+      router={router}
+      theme={{ light: baseLightTheme, dark: baseDarkTheme }}
```

Comment:
Do we need this `property` here? If we need it, I'd expect it to set the primary color to one that matches the logo.

I also don't think we should promote anything other than css vars themes. The only place where we mention or use legacy theme should be where we document specifically them.

Reply:
You mean the `theme` property?
There's actually some conflict when using the CSS vars theme in the docs examples which is why I'm using `theme={{ light: baseLightTheme, dark: baseDarkTheme }}`... Maybe the class names for the iframes can be given a different name or something and that will work, I'll check out what we can do there.

Reply:
> You mean the theme property?

Yes, can't this demo just rely on the default theme?

Reply:
It's using the default CSS variables theme now including in docs examples, but had to add a `window` prop to use the docs demo window instead of the whole page's.
Let me know if you think any alternative would be better and we can change it, even if in a separate PR.